exonum-java-binding/common/src/test/java/com/exonum/binding/common/hash/HashTestUtils.java

```diff
@@ -54,6 +55,7 @@
  * @author Dimitris Andreou
  * @author Kurt Alfred Kluever
  */
+@CiOnly // The SUT and tests are imported and rarely, if ever, change (see 4725ab9e)
```

Comment:
Is it really required here? There aren't tests in this class

Reply:
Absolutely not — thanks!